### PR TITLE
Revert #1172 feature

### PR DESCRIPTION
**Pull request by raman-m**
## Related to
- #1172 
- #1805 

## Proposed Changes
  - Revert #1172 
  - Take into account teams [discussion](../pull/1805#issuecomment-1827529153) and quick [review](../pull/1805#pullrequestreview-1750096514)
  - Create new release for [Oct'23](../milestone/1) milestone without the feature above

### CODE REVIEW

**Comment by raman-m** [comment]
@ggnaegi  FYI

File: src/Ocelot/Cache/CacheKeyGenerator.cs [hunk 1]
```diff
@@ -1,43 +1,20 @@
...
-                        .Append(header);
-                }
-            }
-
-            if (!downstreamRequest.HasContent)
+            var downStreamUrlKeyBuilder = new StringBuilder($"{downstreamRequest.Method}-{downstreamRequest.OriginalString}");
+            if (downstreamRequest.Content != null)
             {
-                return MD5Helper.GenerateMd5(builder.ToString());
+                var requestContentString = Task.Run(async () => await downstreamRequest.Content.ReadAsStringAsync()).Result;
```
Comment (RaynaldM) [inline src/Ocelot/Cache/CacheKeyGenerator.cs:12]: I'm not sure it is a good practice to get result like this

**Review by RaynaldM — APPROVED** [review]

File: src/Ocelot/Cache/CacheKeyGenerator.cs (hunk 1 above)
Comment (ggnaegi) [reply src/Ocelot/Cache/CacheKeyGenerator.cs:12]: No, there is some refactoring needed here but that's some code back in 2019...

Comment (raman-m) [reply src/Ocelot/Cache/CacheKeyGenerator.cs:12]: We will fix the next 2nd round while feature redevelopment.
I guess it is not major issue now.

**Comment by raman-m** [comment]
@ggnaegi go?

**Comment by raman-m** [comment]
@ggnaegi Today is release day!

File: src/Ocelot/Cache/CacheKeyGenerator.cs (hunk 1 above)
Comment (raman-m) [reply src/Ocelot/Cache/CacheKeyGenerator.cs:12]: @RaynaldM Not our team, not even the author, but Mr. Ali added this staff in 2019. 😉

**Pull request merged**